examples/keyvalue/server/handlers.go

```diff
@@ -37,9 +38,9 @@ type YarpcHandler struct {
 	items map[string]string
 }
 
-func NewYarpcThriftHandler(svc service.Host) ([]transport.Registrant, error) {
+func NewYarpcThriftHandler(svc service.Host) ([]rpc.Registrant, error) {
 	handler := &YarpcHandler{items: map[string]string{}}
-	return kvs.New(handler), nil
+	return rpc.WrapRegistrants(svc, kvs.New(handler)), nil
```

Comment:
Why force the user to wrap in their code? This seems ugly. Can we not just wrap it and have them pass us `transport.Registrants` and wrap it transparently?